appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1166,6 +1201,119 @@ definitions:
     required:
       - status
       - _links
+  PagedDayActivity:
+    type: object
+    properties:
+      _links:
+        allOf:
+          - $ref: '#/definitions/Curies'
+          - $ref: '#/definitions/SelfLink'
+        properties:
+          next:
+            properties:
+              href:
+                type: string
+                description: Link to next page, if available
+            required:
+              - href
+          self:
+            properties:
+              href:
+                type: string
+                description: Identifies this same resource
+            required:
+              - href
+          prev:
+            properties:
+              href:
+                type: string
+                description: Link to previous page, if available
+            required:
+              - href
+        required:
+          - next
+          - self
+          - prev
+      _embedded:
+        properties:
+          "yona:dayActivities":
+            type: array
+            items:
+              $ref: '#/definitions/DayActivity'
+      page:
+        description: Information about the current page
+        properties:
+          size:
+            type: string
+            description: The number of elements on the current page
+          totalElements:
+            type: string
+            description: The total number of elements
+            example: 300
+          totalPages:
+            type: string
+            description: The total number of pages
+            example: 15
+          number:
+            type: string
+            description: The current page number.
+            example: 1
+        required:
+          - size
+          - totalElements
+          - totalPages
+          - number
+    required:
+      - _links
+      - _embedded
+      - page
+  DayActivity:
+    allOf:
+      - $ref: '#/definitions/IntervalActivity'
+    properties:
+      date:
+        type: string
+        example: 2016-03-16
+        description: The day date
+      goalAccomplished:
```

Comment:
If we ensure the ``totalMinutesBeyondGoal`` is always > 0 when the goal is not accomplished, we don't need this boolean.